routing/localchans/manager.go

```diff
@@ -104,3 +115,34 @@ func (r *Manager) UpdatePolicy(newSchema routing.ChannelPolicy,
 
 	return nil
 }
+
+// checkMaxHtlc errors if the user tries to update their channel policy with
+// a max HTLC value that is too high or too low for a given list of channels.
+func (r *Manager) checkMaxHtlc(chanPoint wire.OutPoint,
+	maxHtlcRequested lnwire.MilliSatoshi) error {
+
+	ch, err := r.Db.FetchChannel(chanPoint)
+	if err != nil {
+		return err
+	}
+
+	if maxHtlcRequested < ch.LocalChanCfg.MinHTLC {
+		return fmt.Errorf("max htlc size of %v msat is below "+
+			"min HTLC policy of %v msat for channel %v",
+			maxHtlcRequested, ch.LocalChanCfg.MinHTLC,
+			ch.FundingOutpoint)
+	}
+
+	// The max HTLC policy field must be less than or equal to the
```

Comment:
Could combine this and the if statement above into a single switch statement.

Reply:
Code rewritten